.travis.yml

```diff
@@ -18,7 +18,7 @@ sudo: required
 dist: trusty
 
 jdk:
-  - oraclejdk8
+  - oraclejdk11
```

Comment:
@jpechane, as per the discussion in the original PR I thought we wanted to keep the build running on 8 for the time being?

Reply:
@jpechane ping ^

Reply:
@gunnarmorling Done